README.md

```diff
@@ -32,7 +32,10 @@ Bevy is still in the _very_ early stages of development. APIs can and will chang
 * **[Bevy Rust API Docs](https://docs.rs/bevy):** Bevy's Rust API docs, which are automatically generated from the doc comments in this repo.
 
 ## Community
+* Before contributing or participating in discussions with the community, <br>
+it's a good thing to get familiar with our **[Code of Conduct](https://github.com/bevyengine/bevy/blob/master/CODE_OF_CONDUCT.md)**
 * **[Discord Server](https://discord.gg/gMUk5Ph):** Bevy's official discord server.
+* **[Subreddit](https://reddit.com/r/bevy):** Bevy's official subreddit.
```

Comment:
Can we change these to:
```
[Discord]
[Reddit]
```

i think thats a bit "friendlier"

Reply:
Sure will fix